src/Jeu.php

```diff
@@ -0,0 +1,17 @@
+<?php 
+namespace trivial;
+
+use trivial\Plateau;
+
+class jeu {
+    private $plateau= array();
+
+    public function __construct(){
+        $this->plateau = new Plateau();
+        $this->plateau->creaPlateau();
+    }
+
+    public function getPlateau(){
```

Comment:
Is there a point having a getter instead of making your `$plateau` property public?